src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  font-family: Arial, Helvetica, sans-serif;
+  width: 720px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+  margin: 0 auto;
+  
+  @each $day-week, $position in $days-week {
+    &--start-day--#{$day-week} :nth-child(1) {
+      margin-left: $position;
+    }
+  }
+
+  @for $index from 1 through 31 {
+    & :nth-child(#{$index})::before {
+      content: "#{$index}";
+      display: flex;
+      font-size: 30px;
+      line-height: 100px;
+      justify-content: center;
+    }
+  }
+  
+  @for $length from $length-of-month to 31 {
+    &--month-length--#{$length-of-month} :nth-last-child(31 - #{$length}) {
+      display: none;
+    }
+  }

```

Comment:
try smth like that

```suggestion
  @for $length from 28 to 31 {
    &--month-length--#{$length} :nth-child(n + #{$length + 1}) {
      display: none;
    }
  }

```

Reply:
I understood how it work. When I use :nth-chilld, in brackets should be only result.